.swiftlint.yml

```diff
@@ -1,2 +1,8 @@
 parent_config: https://raw.githubusercontent.com/Automattic/swiftlint-config/0f8ab6388bd8d15a04391825ab125f80cfb90704/.swiftlint.yml
 remote_timeout: 10.0
+
+opt_in_rules:
+  - overridden_super_call
+
+overridden_super_call:
+  severity: error
```

Comment:
After seeing what the feedback is on this PR, I might move the definition upstream in the shared config.